src/main/java/fact/io/hdureader/BinTable.java

```diff
@@ -35,6 +35,12 @@
     final DataInputStream tableDataStream;
     DataInputStream heapDataStream = null;
 
+    private final Header header;
```

Comment:
why not public final?

Reply:
I learned not to expose any variables outside so I did that here, but yeah it is final it can be public. Changed it and removed the getter.